palantirnet/federated-search-react
# FS-17: Dependency security updates

### Jira Ticket

https://palantir.atlassian.net/browse/FS-17

### Description 
- Integration branch with automated security fixes for:
  * macaddress (no longer in `yarn.lock` after react-scripts update; `css-loader` no longer requires it)
  * handlebars
  * fstream

- Updates https://github.com/palantirnet/create-react-app (react-scripts) with new versions of `css-loader`, `postcss-loader`, and `jest` to address security issues in `js-yaml`.
- Updates `eslint` dependency  for `js-yaml` security update.
- Updates `node-sass-chokidar` for `tunnel-agent` security update.
- See additional updates to packages here: https://github.com/palantirnet/federated-search-react/pull/45/files

## Testing

- `yarn install`
- `yarn why handlebars`
- `yarn why fstream`
- `yarn why js-yaml`
- `yarn why tunnel-agent`
- Notice none of the installed versions are affected by security vulnerabilities.
- `yarn start`
- Verify React app development functions as expected using this branch.

## To do

- [ ] Create a release for `@palantirnet/react-scripts` and update this PR.
